spacy/language.py

```diff
@@ -1991,6 +2002,45 @@ def to_disk(
         serializers["vocab"] = lambda p: self.vocab.to_disk(p, exclude=exclude)
         util.to_disk(path, serializers, exclude)
 
+    @staticmethod
+    def _resolve_component_status(
+        disable: Iterable[str], enable: Iterable[str], pipe_names: Collection[str]
+    ) -> Tuple[str, ...]:
+        """Derives whether (1) `disable` and `enable` values are consistent and (2)
+        resolves those to a single set of disabled components. `enable` takes precedence
+        over `disable` - so if the former is set, all other components are assumed to be
+        disabled, independently of the value of `disable`. If `enable` is empty, only
+        those component whose names are included in `disable` are assumed to be
+        disabled.
```

Comment:
This doc string is out-of-date with the current functionality that checks compatibility between the `disable` and `enable`